http/headers/timing-allow-origin.json

```diff
@@ -0,0 +1,57 @@
+{
+  "http": {
+    "headers": {
+      "Timing-Allow-Origin": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Timing-Allow-Origin",
```

Comment:
This URL needs to be without "en-US" (we'll add this to the linter validation soon hopefully).
https://developer.mozilla.org/docs/Web/HTTP/Headers/Timing-Allow-Origin